addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/profiles/EnOceanProfileTypes.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.enocean.internal.profiles;
+
+import static org.openhab.binding.enocean.internal.EnOceanBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.CoreItemFactory;
+import org.eclipse.smarthome.core.thing.DefaultSystemChannelTypeProvider;
+import org.eclipse.smarthome.core.thing.profiles.ProfileTypeBuilder;
+import org.eclipse.smarthome.core.thing.profiles.ProfileTypeUID;
+import org.eclipse.smarthome.core.thing.profiles.StateProfileType;
+import org.eclipse.smarthome.core.thing.profiles.TriggerProfileType;
+
+/**
+ * Class holds ProfileTypeUIDs and ProfileTypes of EnOcean profiles
+ *
+ * @author Daniel Weber - Initial contribution
+ */
+public class EnOceanProfileTypes {
+
+    public static final ProfileTypeUID RockerSwitchToPlayPause = new ProfileTypeUID(BINDING_ID,
```

Comment:
As all those are constants, you should change the name to all upper case.

Reply:
removed all profiles => fixed